sim/hunter/dps_hunter/dps_hunter_test.go

```diff
@@ -170,25 +155,32 @@ func TestSV(t *testing.T) {
 			EPReferenceStat: proto.Stat_StatAttackPower,
 			StatsToWeigh:    Stats,
 		},
-		{
-			Class:      proto.Class_ClassHunter,
-			Phase:      5,
-			Level:      60,
-			Race:       proto.Race_RaceOrc,
-			OtherRaces: []proto.Race{proto.Race_RaceDwarf},
+	}))
+}
 
-			Talents:          Phase4WeaveTalents,
-			GearSet:          core.GetGearSet("../../../ui/hunter/gear_sets", "p5_weave"),
-			Rotation:         core.GetAplRotation("../../../ui/hunter/apls", "p5_weave"),
-			Buffs:            core.FullBuffsPhase5,
-			Consumes:         Phase4Consumes,
-			SpecOptions:      core.SpecOptionsCombo{Label: "Weave", SpecOptions: Phase4PlayerOptions},
-			StartingDistance: core.MinRangedAttackRange,
+func TestRangedItemSets(t *testing.T) {
+	core.RunTestSuite(t, t.Name(), core.FullCharacterTestSuiteGenerator([]core.CharacterSuiteConfig{
+		core.GetTestBuildFromJSON(proto.Class_ClassHunter, 4, 60, buildsDir, "p4_ranged", ItemFilters, proto.Stat_StatAgility, Stats),
+		core.GetTestBuildFromJSON(proto.Class_ClassHunter, 5, 60, buildsDir, "p5_ranged", ItemFilters, proto.Stat_StatAgility, Stats),
+		core.GetTestBuildFromJSON(proto.Class_ClassHunter, 6, 60, buildsDir, "p6_ranged", ItemFilters, proto.Stat_StatAgility, Stats),
+		core.GetTestBuildFromJSON(proto.Class_ClassHunter, 7, 60, buildsDir, "p7_ranged", ItemFilters, proto.Stat_StatAgility, Stats),
+	}))
+}
 
-			ItemFilter:      ItemFilters,
-			EPReferenceStat: proto.Stat_StatAttackPower,
-			StatsToWeigh:    Stats,
-		},
+func Test2hItemSets(t *testing.T) {
+	core.RunTestSuite(t, t.Name(), core.FullCharacterTestSuiteGenerator([]core.CharacterSuiteConfig{
+		core.GetTestBuildFromJSON(proto.Class_ClassHunter, 5, 60, buildsDir, "p5_melee_2h", ItemFilters, proto.Stat_StatAgility, Stats),
+		core.GetTestBuildFromJSON(proto.Class_ClassHunter, 6, 60, buildsDir, "p6_melee_2h", ItemFilters, proto.Stat_StatAgility, Stats),
+		core.GetTestBuildFromJSON(proto.Class_ClassHunter, 7, 60, buildsDir, "p7_melee_2h", ItemFilters, proto.Stat_StatAgility, Stats),
+	}))
+}
+
+func TestDwItemSets(t *testing.T) {
```

Comment:
🔨 Naming these something like `TestRangedHunter` `Test2HMeleeHunter` `TestDWMeleeHunter` would fit the conventions we normally use a bit better!